scripts/linux/runE2ETest.sh

```diff
@@ -1040,6 +1045,7 @@ function usage() {
     echo ' -metricsEndpointsCSV              Optional csv of exposed endpoints for which to scrape metrics.'
     echo ' -metricsScrapeFrequencyInSecs     Optional frequency at which the MetricsCollector module will scrape metrics from the exposed metrics endpoints. Default is 300 seconds.'
     echo ' -metricsUploadTarget              Optional upload target for metrics. Valid values are AzureLogAnalytics or IoTHub. Default is AzureLogAnalytics.'
+    echo ' -hostPlatform                     Describes the host OS and cpu architecture.'
```

Comment:
In general, it is possible to get host platform for linux and RPi by script, instead of passing in.  It is required to pass in for Windows since it has many different skus.

Reply:
I would rather keep it consistent. There is also the benefit of matching the names directly to the pipeline.